### PR TITLE
Renaming configDir for removed packages

### PR DESCRIPTION
**Pull request by darkdef**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #2 

Renaming dir with config for removed packages to packageName.removed

### CODE REVIEW

File: src/ComposerEventHandler.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public function onPostAutoloadDump(Event $event): void
         $packages = $composer->getRepositoryManager()->getLocalRepository()->getPackages();
 
         foreach ($this->removals as $packageName) {
-            $this->removePackageConfig($packageName);
+            $this->removePackageConfig($packageName, $outputDirectory);
```
Comment (samdark) [inline src/ComposerEventHandler.php:89]:
```suggestion
            $this->markPackageConfigAsRemoved($packageName, $outputDirectory);
```

**Review by samdark — APPROVED** [review]
Looks good except method name.

File: src/ComposerEventHandler.php (hunk 1 above)
Comment (darkdef) [reply src/ComposerEventHandler.php]: ok

File: tests/Integration/ComposerEventHandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+TXT
+        );
+        $this->execComposer('install');
+    }
+
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+        $workingDirectory = $this->getWorkingDirectory();
+        $this->exec("rm -rf $workingDirectory/*");
```
Comment (samdark) [inline tests/Integration/ComposerEventHandlerTest.php:53]: Won't work with Windows. Let's require-dev yiisoft/files and use `FileHelper::removeDirectory()`.

**Review by samdark — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**